lib/catalog_of_my_things/book.rb

```diff
@@ -0,0 +1,17 @@
+require_relative './item'
+
+class Book < Item
+  attr_accessor :publisher, :cover_state
+
+  def initialize(publish_date, publisher, cover_state)
```

Comment:
### Optional changes

- [ ] We should consider including in your constructor the id and archived as optional parameters. E.g:

`def initialize(publish_date, publisher, cover_state, id = null, archived: false)`

Reply:
Done! Thanks for the suggestion. 🙂